workshops/intro-to-ML/README.md

```diff
@@ -0,0 +1,1694 @@
+---
+name: 'Intro To ML'
+description: 'Making an Machine Learning model to predict relation between 2 number sets '
+author: '@Agrim-Bansal'
+---
+
+
+
+
+
+# Introduction
+
+
+Today We'll make a machine learning model where it learns the relationship between two sets of numbers.
```

Comment:
let's make this a bit more exciting

Reply:
Could You please give some suggestions for it

Reply:
take a look at some others on the site to get an idea, sorry im busy rn